packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 15.0.3
```

Comment:
Isn't this a breaking change for Java? I believe the failure is only when using the Java code with Kotlin, in which case many people will have currently working code with 15.x that will suddenly have the wrong type.